src/datacatalogtordf/resource.py

```diff
@@ -46,7 +48,7 @@ class Resource(ABC):
         modification_date (Date): Most recent date on which the item was changed,\
             updated or modified.
         language (List[str]): A list of links to languages of the item.
-        publisher (URI):  A URI uniquely identifying the publisher of the resource
+        publisher (Any):  A URI uniquely identifying the publisher of the resource
```

Comment:
Kanskje bare litt mismatch mellom teksten (som sier at det er URI) og datatypen Any - antar at man bruker Any her fordi det er et objekt eller BNode?

Reply:
Korrekt. Må bruke Any for å ikkje introdusere breaking change: Det skal vere mogleg både å sette til URI og til Agent-objekt